### DISCUSSION TITLE
WYSIWYG field to gutenberg editor field

### DISCUSSION THREAD

**Discussion by Nyam** [WordPress/gutenberg#37521] [Q&A]
Hello! Please, help me to add gutenberg editor field to my custom form. I need to change my WYSIWYG field to gutenberg editor field.

Thank you

**Comment by dmsnell** [comment]
Hi @Nyam. It's not super easy to swap out a simple entry field to Gutenberg but you can try out the [isolated block editor](https://github.com/Automattic/isolated-block-editor) and see if it can help you. There are different contexts in which that can add a Gutenberg editor for you, some from the hosting side and some from the browser-extension side.
